src/shared/components/Select/styles.ts

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import Colors from 'shared/colors';
+
+type SelectProps = {
+  openedSelect: boolean;
```

Comment:
if it's boolean let's better call it `isOpenedSelect `

Reply:
done